tests/main/lxd-no-fuse/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Check that we give a useful error when fuse is missing in lxd
+
+# we just need a single system to verify this
+systems: [ubuntu-18.04-64]
```

Comment:
Can you please mirror changes to the LXD tests recently introduced here? Alternatively delegate this to me, so that we do proper cleanup.

Reply:
I pick option (2), please push whatever is needed for proper cleanup.

Reply:
I didn't notice this is just for Ubuntu 18.04, this is covered by the general fixes I did that affect the entire machine so the test doesn't need anything extra. LGTM